backend/auth/auth.go

```diff
@@ -57,86 +58,96 @@ func (jwt *JWT) VerifierHandler() func(http.Handler) http.Handler {
 	return jwtauth.Verifier(jwt.TokenAuth)
 }
 
+// parse uri => type, data, ok
+func parseURI(uri string) (int, []string, bool) {
```

Comment:
This function is not cool (because the return paramter `int` is not clear what it is it will make ppl confuse also we can use multiple return function instead of array